src/connectivities.jl

```diff
@@ -11,7 +11,7 @@ of type `PL`. Indices are taken from a global vector of [`Point`](@ref).
 Connectivity objects are constructed with the [`connect`](@ref) function.
 """
 struct Connectivity{PL<:Polytope,N}
-  indices::NTuple{N,Int}
+  indices::SVector{N,Int}
```

Comment:
Thinking about this again, I am bit concerned that the constructor accepts a NTuple and the field is SVector. When users call indices(connec) they will get back a SVector even though they inserted a NTuple. Maybe we should preserve NTuple here? Let's discuss over Zulip what is the best way forward...